packages/squiggle-lang/__tests__/library/distOperations_test.ts

```diff
@@ -11,4 +11,23 @@ describe("Dist operations", () => {
       "Error(Distribution Math Error: Operation returned complex result)"
     );
   });
+
+  testEvalToBe("sum([2,Dist.make(2),Dist.make(4)])", "PointMass(8)");
```

Comment:
It might be better for me to make a new dist_test.ts file and put these tests there.